### ISSUE TITLE
Switch Orientation.

### ISSUE THREAD

**Issue by asciireign** [makerdiary/python-keyboard#3]
I'm into mechanical keyboards for a while.
I stumbled upon your project from an reddit/r/mechanicalkeyboard post
and already ordered a pcb from you even though your layout option
is with the basic 60 percent isn't my cup of tea. I usually prefer all my boards to have [tsangan layout](https://www.1upkeyboards.com/wp-content/uploads/2019/01/ANSI-Plate-Layouts.jpg).
only to find that you made one big mistake by not knowing about it.

Your switches are facing the wrong way and therefore will have issues with cherry profile keycaps.
Those keycaps are very common in the keyboard scene and the keys in both
the qwe and asd row will hit the housing of the switch and make an unwanted sound.
This is an issue even without additional leds.

I do know that you are quite far in your design process but this would prevent
quite a lot of people from using this for more as a tinker project.

Sources:
me owning a hotswap pcb with that issue and using silent switches on those rows made them very unsilent.
[Youtube video of Wildcat explaining the issue](https://www.youtube.com/watch?v=Bh93sXRh4x4)
[Reddit post about the plank rev6 having that issue](https://www.reddit.com/r/olkb/comments/8h2ls2/planck_rev_6_has_northfacing_switch_orientation/)
[Reddit post about switch washers sold by novelkeys and olkb specifically to fight that issue](https://www.reddit.com/r/MechanicalKeyboards/comments/9jdmi3/novelkeys_north_facing_keyswitch_washers_review/)

In my opinion the way to fix this would be to orient all switches so the leds are facing the south expect:
the switches in the bottom row to the north so that the don't shine towards the user
the switches next to the usb port so you don't have an issue with the clearance.

**Comment by asciireign** [comment]
Yes it is. There's [another picture](https://i.imgur.com/K7uAwco.jpg)

**Comment by xiongyihui** [comment]
The switches that we are going to use don't have this issue. However, some switches indeed have the issue.

![image](https://user-images.githubusercontent.com/948283/91940488-17af5f80-ed2a-11ea-93bc-a7b3f421a6ca.png)

The pink switch is the one we choose to use.

![image](https://user-images.githubusercontent.com/948283/91941721-3a427800-ed2c-11ea-9664-93e6694e2204.png)

**Comment by asciireign** [comment]
Thank you for understanding the problem.
I do know that it is some work to rearrangebut maybe in revision 2 you could
rotate the qwe and asd rows to make all switches, in the more common design, usable with this pcb.
(or all expect the switches surrounding the usb port)

Looking forward to receive my proto even if i cant use my gateron ink switches with it.

**Comment by CVYY39** [comment]
This definitely limits the keycaps & switches that can be used. I think it is worth fixing if there is another revision.

**Comment by CVYY39** [comment]
Front-side LEDs (as opposed to rear) would also enable the use of shine-through keys with front-mounted legends.
